### PR TITLE
Bug/284781  bugfixes owner

### PR DESCRIPTION
**Pull request by whitewaterdesign**
- separates country from address on Record Created screen - https://dangerous-dogs-index-2616278c1d9a.herokuapp.com/v7/cdo/create/confirmation
- removes line separating Country from address on Confirm all Details screen - https://dangerous-dogs-index-2616278c1d9a.herokuapp.com/v8/cdo/create/confirm-details

### CODE REVIEW

File: app/lib/format-helpers.js [hunk 1]
```diff
@@ -1,19 +1,48 @@
...
+const formatAddress = (address, hideCountry = false) => {
   if (!address) {
     return null
   }
 
   const parts = []
 
   Object.keys(address).forEach(key => {
-    if (address[key]) {
+    const checkCountry = key !== 'country' || hideCountry === false
```
Comment (jbarnsley10) [inline app/lib/format-helpers.js:21]: I find this var name a bit confusing. I would reckon it should be 'includeField' as it actually relates to fields other than country.

File: app/views/cdo/create/confirm-all-details.njk [hunk 2]
```diff
@@ -58,14 +58,12 @@
...
             </div>
-            <div class="govuk-summary-list__row govuk-summary-list__row--no-actions">
-              <dt class="govuk-summary-list__key">
-                Country
-              </dt>
-              <dd class="govuk-summary-list__value">
-                {{ model.summary.owner.country }}
-              </dd>
-            </div>
+            <dt class="govuk-summary-list__key">
```
Comment (jbarnsley10) [inline app/views/cdo/create/confirm-all-details.njk:61]: I assume this is correct based on the prototype - with horizontal border (if relevant)?

**Review by jbarnsley10 — CHANGES_REQUESTED** [review]
LGTM - just that variable name please

File: app/views/cdo/create/confirm-all-details.njk (hunk 2 above)
Comment (whitewaterdesign) [reply app/views/cdo/create/confirm-all-details.njk:61]: this is the prototype:
https://dangerous-dogs-index-2616278c1d9a.herokuapp.com/v8/cdo/create/confirm-details
```
<div class="govuk-summary-list__row">
          <dt class="govuk-summary-list__key">
            Address
          </dt>
          <dd class="govuk-summary-list__value">
            12 Primrose Road<br>Southampton<br>SO16 3GP
          </dd>

            <dd class="govuk-summary-list__actions">

                <a class="govuk-link govuk-link--no-visited-state" href="#">Change<span class="govuk-visually-hidden"> address (Owner details)</span></a>
            </dd>
        </div>

          <dt class="govuk-summary-list__key govuk-!-padding-top-0">
            Country
          </dt>
          <dd class="govuk-summary-list__value govuk-!-padding-top-0">
            England
          </dd>```

**Review by jbarnsley10 — APPROVED** [review]

**Pull request merged**